profiler.sh

```diff
@@ -93,7 +93,9 @@ check_if_terminated() {
 
 fdtransfer() {
     if [ "$USE_FDTRANSFER" = "true" ]; then
-        "$FDTRANSFER" "$PID"
+        FDTRANSFER_PATH="@async-profiler-$(od -An -N3 -i /dev/random | xargs)"
```

Comment:
`od -An -N3 -i /dev/random | xargs` is the most generic way I found to get a random number, stripped of spaces. (`$RANDOM` works in bash but not sh)

Reply:
Since the script uses `/bin/sh` it cannot use `-o pipefail`, and thus if `od` or `xargs` are missing the script will continue instead of getting an error. I suppose this can be worked around by doing this without a subshell...